### DISCUSSION TITLE
Fehlende Werte

### DISCUSSION THREAD

**Discussion by gmbh07** [dominikamann/oekofen-pellematic-compact#74] [Ideas]
Hallo,

wenn ich eine Abfrage mit der URL mache erhalte ich folgende Zeilen und noch viele weitere, ich Finde aber L_existing_boiler":742, nicht bei den Sensoren, kann ich die Config irgendwie um diesen Wert erweitern? Es handelt sich wir um einen normalen Holzofen.

{
 "system":{
  "L_ambient":163, 
  "L_errors":0, 
  "L_usb_stick":0, 
  **"L_existing_boiler":742,** 
  "mode":1
 },
 "weather":{
  "L_temp":160, 
  "L_clouds":71,

**Comment by dominikamann** [comment]
Hi,

yes this is possible. Please tell me the correct unit and factor then I will extend the integration. You can find this out by using "?" in the REST-URL.

**Comment by gmbh07** [comment]
Good Moring,

thanks for your support. Here are the information:

 "system":{
  "system_info":"system global variables",
  "L_ambient":{"val":89, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Außentemperatur"}, 
  "L_errors":{"val":0, "factor":1, "min":-32768, "max":32767, "text":"Fehler"}, 
  "L_usb_stick":{"val":0, "format":"0:Aus|1:Ein", "text":"Usb Stick erkannt"}, 
  "L_existing_boiler":{"val":272, "unit":"°C", "factor":0.1, "min":-32768, "max":32767, "text":"Bestehender Kessel"}, 
  "mode":{"val":1, "format":"0:Aus|1:Auto|2:Warmwasser", "text":"Betriebsart"}

**Comment by dominikamann** [comment]
Thanks I will integrate it once I'm at home. Currently I'm in Spain without access to a PC. I will let you know once it is done 👍

**Comment by dominikamann** [comment]
@gmbh07  - Implemented and released see https://github.com/dominikamann/oekofen-pellematic-compact/releases/tag/v3.3.4
